src/WarcraftLegacies.Source/Setup/Spells/GoblinSpellSetup.cs

```diff
@@ -56,9 +56,6 @@ public static void Setup()
       {
         Duration = 10
       });
-
-      PassiveAbilityManager.Register(new ProvidesIncome(Constants.UNIT_NBOT_GOBLIN_PRIVATEER_NEUTRAL_GOBLIN, -2));
```

Comment:
Is this right? Are these units removed or do they not have that gold drain ability anymore?

Reply:
Yeah so it looks like these units indeed aren't used anymore. It's a good cleanup but we should probably just delete the entire unit.